llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -99,6 +99,20 @@ void ModuleShaderFlags::updateFunctionFlags(ComputedShaderFlags &CSF,
     }
   }
 
+  if (!CSF.Int64Ops)
+    CSF.Int64Ops =
+        I.getType()->isIntegerTy() && I.getType()->getScalarSizeInBits() == 64;
```

Comment:
Can `I.getType()->isIntegerTy(64);` be used here - as well as on line 108 - instead?

Reply:
Yes. I didn't realize that was a valid overload.